sdk/metric/periodic_reader_test.go

```diff
@@ -329,6 +329,52 @@ func TestPeriodicReaderFlushesPending(t *testing.T) {
 		_ = r.Shutdown(context.Background())
 	})
 
+	t.Run("ForceFlush timeout on producer", func(t *testing.T) {
```

Comment:
These tests seem to copy how Shutdown is tested. Does it make sense to refactor the shared logic/setup into a reusable function?

Reply:
I was considering it but I did not see a lot of benefits in it as it gave an additional layer of indirection which made the test less readable.

I also try to follow the [Rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))

Reply:
Yeah, I'll give it a go. Thanks :)